src/main/scala/Centis.scala

```diff
@@ -78,4 +78,20 @@ object Centis {
   def ofSeconds(s: Int) = Centis(100 * s)
   def ofMillis(i: Int)  = Centis((if (i > 0) i + 5 else i - 4) / 10)
   def ofMillis(l: Long) = Centis((if (l > 0) l + 5 else l - 4) / 10)
+
+  def withActionStrs(
+      times: Vector[Centis],
+      actionStrs: ActionStrs,
+      offset: Int = 0 // startPlayer.hashCode - 1
```

Comment:
suggest to change the name of this variable from offset to startingPlayer as that it more meaningful and then change calculation based on that? This is only currently used in lila once.